src/decoration.cpp

```diff
@@ -615,6 +615,9 @@ void Decoration::redrawPixmap() {
                     );
                     titleWidth -= tabScheme.outer_width;
                 }
+                drawText(pix, gc, tabScheme.title_font->data(), tabScheme.title_color(),
+                         tabGeo.tl() + Point2D { tabPadLeft, (int)s.title_height()},
+                         tabClient->title_(), titleWidth - tabPadLeft);
                 XSetForeground(display, gc, get_client_color(tabScheme.outer_color));
                 XFillRectangles(display, pix, gc, &borderRects.front(), borderRects.size());
```

Comment:
It might be more appropriate to put the drawText call after these two lines?

Reply:
Yes, the draw text should be after that, otherwise the title might be covered for plain X11 fonts; hopefully the CI will discover this issue :-). (There is something special about the drawing of Xft fonts, and this is why the issue is probably not visible in your screenshot)

Reply:
Updated the pr.